src/main/java/CarRacingGame/Cars.java

```diff
@@ -0,0 +1,78 @@
+package CarRacingGame;
+
+import java.util.*;
+
+public class Cars implements Iterable<Car> {
+    private final List<Car> cars;
+
+    public Cars(List<Car> cars) {
+        this.cars = cars;
+    }
+
+    public Cars(String carsNames) {
+        List<String> splitNames = Arrays.asList(carsNames.split(","));
+        List<Car> result = new ArrayList<>();
+
+        splitNames.forEach(name -> {
+            try {
+                Car verifyCar = new Car(name);
+                result.add(verifyCar);
+            } catch (IllegalArgumentException e) {
+                System.out.println(e.getMessage());
+            }
+        });
+
+        cars = result;
+    }
+
+    public void moveWithCondition() {
+        cars.forEach(car -> {
+            if (InputUI.moveCondition()) {
+                car.move();
+            }
+        });
```

Comment:
```suggestion
        cars.filer
        .forEach(car -> {
            if (InputUI.moveCondition()) {
                car.move();
            }
        });
```
```suggestion
        cars.stream()
            .filter(it -> InputUI.moveCondition())
            .forEach(Car::move);
```
stream 사용한다면 하나의 메서드에서 하나의 로직만